### DISCUSSION TITLE
How can I map api routes to a different path?

### DISCUSSION THREAD

**Discussion by salazarm** [vercel/next.js#17360] [Help] [answered]
The site I'm developing sits behind an ngnix server that snipes /api/ traffic and redirects it else where. Is there a way to have it go somewhere else, say /_api/?

I tried doing this using rewrites in my next.config.js:
```
{
...
async rewrites() {
        return [
          {
            source: '/_api/:slug*',
            destination: '/api/:slug*',
          },
        ]
      },
...
}
```

**Comment by salazarm** [comment]
Looks like its hardcoded in =\ https://github.com/vercel/next.js/blob/c351f6154bcc31b2261497a925ebc67a5ef4e076/packages/next/next-server/server/next-server.ts#L654

**Comment by jamesmosier** [comment, marked as answer]
Correct there is no way to change this natively in Next.
